### PR TITLE
add travis

### PR DESCRIPTION
**Pull request by krubenok**
it does the automatic deploy things!  

Followed this loosely along with another tab I closed (oops) https://medium.com/@bezgachev/6-simple-steps-to-automatically-test-and-deploy-your-javascript-app-to-github-pages-c4c32a34bcb1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm run build
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $github_token
```
Comment (pierreTklein) [inline .travis.yml]: Do we have a `github_token`?

Comment (pierreTklein) [reply .travis.yml]: or does this populate automatically

Comment (krubenok) [reply .travis.yml]: Yeah, it's in travis settings since it's a GitHub access token that has write access to master

Comment (pierreTklein) [reply .travis.yml]: Great

Comment (loreina) [reply .travis.yml]: we used an env variable so the token isn't public

**Pull request merged**